src/presenter/event-presenter.js

```diff
@@ -0,0 +1,35 @@
+import { render } from '../render';
+import EventEditView from '../view/main/event-edit-view';
+import EventItemView from '../view/main/event-item-view';
+import EventListView from '../view/main/event-list-view';
+import EventView from '../view/main/event-view';
+import MainView from '../view/main/main-view';
+import SortView from '../view/main/sort-view';
+import TripView from '../view/main/trip-view';
+
+export default class EventPresenter {
+  mainComponent = new MainView();
+  tripComponent = new TripView();
+  sortComponent = new SortView();
+  eventListComponet = new EventListView();
+  editComponent = new EventEditView();
+
+  constructor ({mainContainer}){
+    this.mainContainer = mainContainer;
+  }
```

Comment:
Нет смысла добавлять лишние слова, усложнять читаемость без особой необходимости.
У нас всего один контейнер в компоненте, поэтому и без `main` понятно
```suggestion
  constructor ({сontainer}){
    this.сontainer = сontainer;
  }
```